Cloud: update release notes for 1122 (#11396)
pingcap/docs: tidb-cloud/release-notes-2022.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.

## November 22, 2022

**General changes**

> * Support directly migrating data from Amazon Aurora MySQL, Amazon Relational Database Service (RDS) MySQL, or self-built MySQL-compatible databases to TiDB Cloud online.

Reviewer: ```suggestion
* Support directly migrating data from Amazon Aurora MySQL, Amazon Relational Database Service (RDS) MySQL, or self-hosted MySQL-compatible databases to TiDB Cloud online.
```

Reviewer: use the same word as in https://github.com/pingcap/docs/pull/11117

> * Support directly migrating data from Amazon Aurora MySQL, Amazon Relational Database Service (RDS) MySQL, or self-built MySQL-compatible databases to TiDB Cloud online.

Reviewer 2: ```suggestion
* Support directly migrating data from Amazon Aurora MySQL, Amazon Relational Database Service (RDS) MySQL, or self-built MySQL-compatible databases to TiDB Cloud online (beta).
```

> * Support directly migrating data from Amazon Aurora MySQL, Amazon Relational Database Service (RDS) MySQL, or self-hosted MySQL-compatible databases to TiDB Cloud online.

Author: ```suggestion
* Support directly migrating data from Amazon Aurora MySQL, Amazon Relational Database Service (RDS) MySQL, or self-hosted MySQL-compatible databases to TiDB Cloud online (beta).
```

>     Previously, you needed to stop your business and import data offline, or use third-party tools to migrate data to TiDB Cloud, which was complicated. Now, with the **Data Migration** feature, you only need to perform operations on your TiDB Cloud console and securely migrate your data to TiDB Cloud with minimal downtime.

Reviewer 2: ```suggestion
    Previously, you needed to stop your cluster and import data offline, or use third-party tools to migrate data to TiDB Cloud, which was complicated. Now, with the **Data Migration** feature, you only need to perform operations on your TiDB Cloud console and securely migrate your data to TiDB Cloud with minimal downtime.
```

Reviewer: the upstream business might be just on MySQL databases instead of a MySQL cluster

>     Previously, you needed to stop your business and import data offline, or use third-party tools to migrate data to TiDB Cloud, which was complicated. Now, with the **Data Migration** feature, you only need to perform operations on your TiDB Cloud console and securely migrate your data to TiDB Cloud with minimal downtime.

Reviewer: ```suggestion
    Previously, you needed to pause your business and import data offline, or use third-party tools to migrate data to TiDB Cloud, which was complicated. Now, with the **Data Migration** feature, you only need to perform operations on your TiDB Cloud console and securely migrate your data to TiDB Cloud with minimal downtime.
```

>     In addition, Data Migration provides full and incremental data migration capabilities to migrate the existing data and ongoing changes from your data source to TiDB Cloud.

Reviewer: ```suggestion
    In addition, Data Migration provides full and incremental data migration capabilities to migrate both existing data and ongoing changes from your data source to TiDB Cloud.
```

>     Currently, Data Migration is available only for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters and only in AWS Oregon (us-west-2) and AWS Singapore (ap-southeast-1) regions. This feature is still **in beta** and you can create one migration job for free for each organization. To create multiple migration jobs for an organization, you need to [file a ticket](/tidb-cloud/tidb-cloud-support.md).

Reviewer: ```suggestion
    Currently, the Data Migration feature is still **in beta**. It is available only for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters and only in AWS Oregon (us-west-2) and AWS Singapore (ap-southeast-1) regions. You can create one migration job for free for each organization. To create multiple migration jobs for an organization, you need to [file a ticket](/tidb-cloud/tidb-cloud-support.md).
```

>     For detailed information, see [Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration](/tidb-cloud/migration-job.md).

Author: ```suggestion
    For detailed information, see [Migrate MySQL-Compatible Databases to TiDB Cloud Using Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md).
```

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.

## November 22, 2022

**General changes**

> * Support directly migrating data from AWS Aurora, AWS RDS or MySQL-compatible databases to TiDB Cloud

Reviewer 2: ```suggestion
* Support directly migrating data from Amazon Aurora, Amazon Relational Database Service (RDS), or a MySQL-compatible database to TiDB Cloud.
```

```suggestion
* Support directly migrating data from Amazon Aurora, Amazon RDS, or a MySQL-compatible database to TiDB Cloud.
```
